src/main/scala/cromwell/engine/package.scala

```diff
@@ -62,6 +62,10 @@ package object engine {
       val key = SymbolStoreKey(scope, name, iteration = None, input)
       SymbolStoreEntry(key, wdlValue.wdlType, Some(wdlValue))
     }
+
+    def toWorkflowOutputs(t: Traversable[SymbolStoreEntry]): WorkflowOutputs = t.map { e =>
+      e.key.scope + "." + e.key.name -> e.wdlValue.get
```

Comment:
I have a preference toward string interpolating all the things but for things like this it's not a big deal